File: app/src/main/java/com/playdate/app/ui/dashboard/OnFriendSelected.java
package com.playdate.app.ui.dashboard;

public interface OnFriendSelected {
    void OnSingleFriendSelected(String ID,String FreindID);
    void OnFrinedDataClosed();
    void OnSuggestionClosed();
    void OnSuggestionClosed(boolean isFriend,String ID);
}

File: app/src/main/java/com/playdate/app/ui/dashboard/OnFriendSelected.kt
package com.playdate.app.ui.dashboard

interface OnFriendSelected {
    fun onSingleFriendSelected(ID: String?, FreindID: String?)
    fun onFrinedDataClosed()
    fun onSuggestionClosed()
    fun onSuggestionClosed(isFriend: Boolean, ID: String?)
}